src/main/java/commands/EditCommand.java

```diff
@@ -63,7 +78,35 @@ private void editCard(Ui ui, Access access, Storage storage) throws InvalidInput
             storage.saveCards(cards, access.getModule().getModuleName(), access.getChapter().getChapterName());
         } catch (IndexOutOfBoundsException | NullPointerException e) {
             throw new InvalidInputException("The flashcard number needs to be within the range "
-                    + " of the total number of flashcards\n"
+                    + "of the total number of flashcards\n"
+                    + MESSAGE_USAGE);
+        }
+    }
+
+    private void editModule(Ui ui, Access access, Storage storage) throws InvalidInputException, IOException {
+        ModuleList modules = access.getAdmin().getModules();
+        try {
+            Module module = modules.getModule(editIndex);
+            ui.showModuleUnedited(module);
+            module.setModuleName(moduleOrChapter);
+            ui.showModuleEdited(module);
+        } catch (IndexOutOfBoundsException | NullPointerException e) {
+            throw new InvalidInputException("The module number needs to be within the range "
+                    + "of the total number of modules\n"
+                    + MESSAGE_USAGE);
+        }
+    }
+
+    private void editChapter(Ui ui, Access access, Storage storage) throws InvalidInputException, IOException {
+        ChapterList chapters = access.getModule().getChapters();
+        try {
+            Chapter chapter = chapters.getChapter(editIndex);
+            ui.showChapterUnedited(chapter);
+            chapter.setChapterName(moduleOrChapter);
+            ui.showChapterEdited(chapter);
+        } catch (IndexOutOfBoundsException | NullPointerException e) {
+            throw new InvalidInputException("The chapter number needs to be within the range "
+                    + "of the total number of chapters\n"
                     + MESSAGE_USAGE);
```

Comment:
Perhaps MESSAGE_USAGE could be removed as the command is correct but chapter is not found?